### DISCUSSION TITLE
elastic sinks not work

### DISCUSSION THREAD

**Discussion by therus000** [vectordotdev/vector#11877] [Q&A] [answered]
good day
i up vector on kubernetes. i already got there an elasticsearch 
i want to push the kubernetes logs to elastic. and i got error when add sinks of elastic to config map of vector

2022-03-17T11:16:52.083433Z INFO vector::app: Log level is enabled. level="vector=info,codec=info,vrl=info,file_source=info,tower_limit=trace,rdkafka=info,buffers=info"
--
Thu, Mar 17 2022 2:16:52 pm | 2022-03-17T11:16:52.083647Z INFO vector::app: Loading configs. paths=["/etc/vector"]
Thu, Mar 17 2022 2:16:52 pm | 2022-03-17T11:16:52.084116Z ERROR vector::cli: Configuration error. error=while parsing a block mapping, did not find expected key at line 30 column 3

my config map with sinks look like this:
---
# This file has been generated by `helm template vector vector/vector` from vector/templates/configmap.yaml. Please re-run `make generate-kubernetes-manifests` rather than modifying this file manually.
apiVersion: v1
kind: ConfigMap
metadata:
  name: vector
  namespace: vector
  labels:
    app.kubernetes.io/name: vector
    app.kubernetes.io/instance: vector
    app.kubernetes.io/component: Agent
    app.kubernetes.io/version: "0.20.0-distroless-libc"
data:
  agent.yaml: |
    data_dir: /vector-data-dir
    api:
      enabled: true
      address: 127.0.0.1:8686
      playground: false
    sources:
      kubernetes_logs:
        type: kubernetes_logs
      host_metrics:
        filesystem:
          devices:
            excludes: [binfmt_misc]
          filesystems:
            excludes: [binfmt_misc]
          mountPoints:
            excludes: ["*/proc/sys/fs/binfmt_misc"]
        type: host_metrics
      internal_metrics:
        type: internal_metrics
    sinks:
      prom_exporter:
        type: prometheus_exporter
        inputs: [host_metrics, internal_metrics]
        address: 0.0.0.0:9090
      stdout:
         type: console
         inputs: [kubernetes_logs]
         encoding:
           codec: json 
      elastic:
         inputs = [kubernetes_logs]            # only take sampled data
         type = "elasticsearch"
         auth.user: "elastic"
         auth.password: "password"
         host = "https://elastic-host:9200"   # local or external host
         index = "vector-%Y-%m-%d"             # daily indices#

**Comment by spencergilbert** [comment, marked as answer]
Hi! Your configuration is using deprecated and removed options, see: https://vector.dev/docs/reference/configuration/sinks/elasticsearch/#bulk and https://vector.dev/highlights/2021-12-28-0-19-0-upgrade-guide/#elasticsearch-sink-deprecated-fields
